reference/resources/payment_service_definition.yaml

```
@@ -95,6 +95,24 @@ properties:
       - GB
       - DE
 
+  supported_features:
+    type: object
+    description: |-
+      Features supported by the payment definition.
+    properties:
+      payment_tokenization:
```

```suggestion
      payment_method_tokenization:
```

It's not the payment we're tokenizing is it?

---

@ipeluffo Does this need to change in the payment definition?

---

@douglaseggleton , I'll apply this change in `core-api` PR to match @cbetta 's suggestion